### DISCUSSION TITLE
Mediapipe- palm_detection_without_custom_op.tflite  with handedness

### DISCUSSION THREAD

**Discussion by Deepakpy** [PINTO0309/PINTO_model_zoo#296] [Ideas]
I would like to know if it's possible to generate palm_detection_without_custom_op.tflite model which also has handedness as one of the outputs. If so can you please share any reference

**Comment by PINTO0309** [comment]
https://github.com/PINTO0309/PINTO_model_zoo/tree/main/033_Hand_Detection_and_Tracking/30_batchN_post-process_marged

![image](https://user-images.githubusercontent.com/33194443/192566080-12e1cb86-04db-4498-bb6d-375ebb2eb67e.png)
![image](https://user-images.githubusercontent.com/33194443/192566131-8ac740da-f547-45f5-8ebf-64b0343c815a.png)

**Comment by Deepakpy** [comment]
Thanks for the reference, Can you tell me how to convert it to tflite.
 Using onnx_tf.prepare(onnx_model).run() throws error.

**Comment by PINTO0309** [comment]
```
$ pip show onnx-tf

Name: onnx-tf
Version: 1.10.0

$ onnx-tf convert -i hand_landmark_080_1x3x224x224.onnx -o saved_model

/usr/local/lib/python3.8/dist-packages/tensorflow_addons/utils/ensure_tf_install.py:53: UserWarning: Tensorflow Addons supports using Python ops for all Tensorflow versions above or equal to 2.3.0 and strictly below 2.5.0 (nightly versions are not supported). 
 The versions of TensorFlow you are currently using is 2.9.0 and is not supported. 
Some things might work, some things might not.
If you were to encounter a bug, do not file an issue.
If you want to make sure you're using a tested and supported configuration, either change the TensorFlow version or the TensorFlow Addons's version. 
You can find the compatibility matrix in TensorFlow Addon's readme:
https://github.com/tensorflow/addons
  warnings.warn(
2022-09-28 18:29:10,356 - onnx-tf - INFO - Start converting onnx pb to tf saved model
WARNING:absl:Found untraced functions such as gen_tensor_dict while saving (showing 1 of 1). These functions will not be directly callable after loading.
2022-09-28 18:29:14,841 - onnx-tf - INFO - Converting completes successfully.
INFO:onnx-tf:Converting completes successfully.
```

**Comment by Deepakpy** [comment]
Thank you that worked. 
Generally, if you want to convert the model to a int8 quantized or fully quantized model. It is required to pass representative_dataset. In case of this mediapipe model what data can be passed or can it be skipped.

Also can you tell me how to port the functions to Tensorflow when the functions are not supported by it.

**Comment by Deepakpy** [comment]
Mediapipe Hand models in tflite are very slow compared to there onnx counter part. Is there any way it can be made to run faster in tflite version
